telethon/client/uploads.py

```diff
@@ -201,9 +201,11 @@ async def send_file(
                 Optional JPEG thumbnail (for documents). **Telegram will
                 ignore this parameter** unless you pass a ``.jpg`` file!
 
-                The file must also be small in dimensions and in-disk size.
-                Successful thumbnails were files below 20kb and 200x200px.
-                Width/height and dimensions/size ratios may be important.
+                The file must also be small in dimensions and in disk size.
+                Successful thumbnails were files below 20kB and 320x320px.
+                For Telegram to accept a thumbnail, you must provide the
+                file's dimensions using either ``hachoir`` or
+                :tl:`DocumentAttributesVideo`.                
```

Comment:
Maybe better indicate that this should go in the list of `attributes`. Perhaps the examples section could be extended.